.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    "env",
```

Comment:
This is meant to only run on node, so what about setting node as the target of the env preset?

```js
"presets": [
  ["env", {
    "targets": {
      "node": "<version of node you want to support>"
    }
  }],
  "stage-2"
}
```